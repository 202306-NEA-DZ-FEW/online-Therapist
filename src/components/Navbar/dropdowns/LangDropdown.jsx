import Link from "next/link";
import React from "react";

export default function LangDropdown() {
    return (
        <div className='w-96 md:w-32 bg-white font-atkinson border bg-grey p-2 shadow-md rounded-lg'>
            <ul className='py-2 first:pt-0  last:pb-0'>
                <Link href='#' locale='en'>
                    <li
                        className=' py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700'
                        href='#'
                    >
                        English
                    </li>
                </Link>
                <Link href='#' locale='ar'>
                    <li
                        className=' py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700'
                        href='#'
                    >
                        العربية
                    </li>
                </Link>
            </ul>
        </div>
    );
}
