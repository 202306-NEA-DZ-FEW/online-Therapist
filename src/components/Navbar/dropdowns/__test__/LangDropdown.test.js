import renderer from "react-test-renderer";
import LangDropdown from "../LangDropdown";

it("renders correctly", () => {
    const tree = renderer.create(<LangDropdown />).toJSON();
    expect(tree).toMatchSnapshot();
});