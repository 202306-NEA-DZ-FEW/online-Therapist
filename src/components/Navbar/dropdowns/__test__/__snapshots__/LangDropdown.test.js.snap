// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="w-96 md:w-32 bg-white font-atkinson border bg-grey p-2 shadow-md rounded-lg"
>
  <ul
    className="py-2 first:pt-0  last:pb-0"
  >
    <a
      href="#"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <li
        className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
        href="#"
      >
        English
      </li>
    </a>
    <a
      href="#"
      onClick={[Function]}
      onMouseEnter={[Function]}
      onTouchStart={[Function]}
    >
      <li
        className=" py-2 px-3 rounded-lg text-lg text-gray-800 hover:bg-gray-100 focus:outline-none focus:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 dark:hover:text-gray-300 dark:focus:bg-gray-700"
        href="#"
      >
        العربية
      </li>
    </a>
  </ul>
</div>
`;
